fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -69,6 +33,12 @@ def self.available_options
         user ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
 
         [
+          FastlaneCore::ConfigItem.new(key: :live,
```

Comment:
done  - with default value `false` and not overwriting it, basically the `latest_testflight_build_number` - should be deprecated - but should work as a BC shim!